projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.ts

```diff
@@ -45,13 +47,20 @@ export class OpenInAppComponent {
   ) {
     if (data) {
       this.redirectUrl = data.redirectUrl;
+      if (data.appStoreUrl) {
```

Comment:
you can do the same as "redirectUrl": `this.appStoreUrl = data.appStoreUrl` like you did above